hathor/manager.py

```diff
@@ -200,10 +199,8 @@ def __init__(self, reactor: IReactorCore, peer_id: Optional[PeerId] = None, netw
         # List of capabilities of the peer
         if capabilities is not None:
             self.capabilities = capabilities
-        elif enable_sync_v2:
-            self.capabilities = [settings.CAPABILITY_WHITELIST, settings.CAPABILITY_SYNC_V2]
         else:
-            self.capabilities = [settings.CAPABILITY_WHITELIST]
+            self.capabilities = [settings.CAPABILITY_WHITELIST, settings.CAPABILITY_SYNC_VERSION]
```

Comment:
Should we move to a constant `DEFAULT_CAPABILITIES`? In this case, if a list of capabilities is given, the caller can choose to extend the default ones or not.

Reply:
I think we should.

Reply:
Done.

Reply:
Great! Thanks!